src/validations/__tests__/javascript.test.js

```diff
@@ -0,0 +1,84 @@
+import partialRight from 'lodash-es/partialRight';
+
+import javascript from '../javascript';
+
+import validationTest from './validationHelper';
+import testValidatorAcceptance from './testValidatorAcceptance';
+
+const analyzer = {
+  enabledLibraries: [],
+  containsExternalScript: false,
+};
+
+const analyzerWithjQuery = {
+  enabledLibraries: ['jquery'],
+  containsExternalScript: false,
+};
+
+const analyzerWithExternalScript = {
+  enabledLibraries: [],
+  containsExternalScript: true,
+};
+
+describe('javascript validation', () => {
+  test('invalid LHS error followed by comment', () => {
+    validationTest(
```

Comment:
Looks like the tests here are still calling the async `validationTest` method without awaiting or returning their result?

Reply:
Fixed